spec/ParsePushAdapter.spec.js

```diff
@@ -375,6 +405,9 @@ describe('ParsePushAdapter', () => {
           publicKey: 'publicKey',
           privateKey: 'privateKey',
         },
+      },
+      expo: {
+
```

Comment:
What's the purpose of this empty object?

Reply:
There are no _required_ parameters to expo push, but there are several optional ones. AccessToken is highly recommended to setup so noone can send notifications to your devices without one (if they got a hold of your push token for a device), but you don't have to (and early in development, some people don't). 

I figured since earlier in the spec I initialized expo push with a parameter, should try without one later to cover more options. 

see:
https://github.com/expo/expo-server-sdk-node/blob/main/src/ExpoClient.ts#L358